src/styles/index.css

```diff
@@ -99,15 +99,25 @@ body {
   border-bottom: 15px solid transparent;
 }
 
-#user-message-section {
+.user-message-section {
   width: 100%;
   position: static;
-  height: 200px;
+  padding: 2px;
+  margin: 2px;
+  height: auto;
   background: rgba(0, 0, 0, 0.8);
   border: solid 3px #fff;
   margin-bottom: 500px;
 }
 
+.message-collapse-button {
+  padding: 1;
+  color: yellow;
```

Comment:
Using SCSS would be much better so we can have common CSS files for all of our colours, common styles, typography.

Reply:
Sure. will look into that before next PR. :)